src/components/DeleteForm.jsx

```diff
@@ -0,0 +1,30 @@
+import { deleteCommentByCommentId } from "../api";
+
+const DeleteForm = ({ id, setComments, setIsDisabled, setErr }) => {
+
+  const handleDelete = (event) => {
+    event.preventDefault();
+    setErr(null);
+    setIsDisabled(true);
+    deleteCommentByCommentId(id)
+      .then((response) => {
+        setErr(null);
+        setIsDisabled(false);
+        setComments((currComments) => {
```

Comment:
could it be easier to pass a function in here that will re-fetch the comments rather than filtering here? 
I think the functionality is fine, maybe a function called remove comment, this would at least stop you having to pass the setState function this far down

Reply:
Ah, I see, as in I could just import the fetch comments into this component and once the delete has succeeded just call the fetch comments that I'd already used.